query.py

```diff
@@ -1,38 +1,81 @@
-import os
+"""This module implements index querying.
+"""
 import argparse
+import os
+import re
 import shelve
-from math import log2
 from gensim.parsing.porter import PorterStemmer
-from merge_operations import *
+from math import log2
+from merge_operations import or_postings, and_postings, not_postings
+from typing import List, Dict, Tuple, Any
 
 
 class Indexer:
-    root = ""
-    docs = []
-    word_count = []
-    stemmer = None
-    index = None
-
-    def __init__(self, docs, index_path, root="lyrics/"):
+    """Class that implements querying and printing results.
+
+    Attributes:
+        root: Directory where songs lyrics is.
+        docs: List of documents filenames, that is used to get docID.
+        word_count: Length of each document.
+        stemmer: Gensim porter stemmer.
+        index: Index file descriptor.
+
+    """
+    root: str

```

Comment:
WARNING! 
Do not mix class variables and instance variables - it's different (if your class is not dataclass)
More: https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3